ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -225,6 +432,62 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec
+- as: azure-ipi-ovn-ipsec-azurefile-csi-arm-f28
+  cron: 14 9 4 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec-azurefile-csi
+- as: azure-ipi-private-arm-f28
+  cron: 45 18 11 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-azure-ipi-private
+- as: azure-ipi-proxy-tp-arm-f28
```

Comment:
This is a TP job, which does not support y-stream upgrade.

Reply:
Hi @JianLi-RH I'd suggest you take note of comments like these somewhere. They are invaluable. Let's also strive to avoid making the same mistakes in the future."

Reply:
I remember that, but missed some of them when manual check it.

Reply:
I remember that, but missed some of them when manual check it.